ddr/lib/ddr-scanner/dwarf/AixSymbolTableParser.cpp

```diff
@@ -122,11 +120,7 @@ dwarf_init(int fd,
 	/* Call dump to get the symbol table. */
 	if (DW_DLV_OK == ret) {
 		stringstream ss;
-#ifdef J9OS_I5
```

Comment:
You'll need to remove `filepath` and its uses else we'll have a compiler warning that will be treated as fatal and break builds.